test/util/server/testing.go

```diff
@@ -0,0 +1,53 @@
+package server
+
+import (
+	"flag"
+
+	configapi "github.com/openshift/origin/pkg/cmd/server/apis/config"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+func StartConfiguredMaster(masterConfig *configapi.MasterConfig) (string, error) {
+	if v := flag.Lookup("test.v"); v == nil {
+		panic("cannot be used outside of test code")
+	}
+
+	// this is for testing. TODO: update the billion call sites with proper stopCh setup code
```

Comment:
That's nasty :stuck_out_tongue_closed_eyes:  but I'll be cleaning that soon-ish.